Assets/HoloToolkit/UX/Scripts/Collections/ObjectCollection.cs

```diff
@@ -0,0 +1,583 @@
+                    // Calculate radius of each node while we're here
+                    // Then use the packer function to shift them into place
+                    for (int i = 0; i < NodeList.Count; i++)
+                    {
+                        newPos = ScatterMapping (nodeGrid[i], Radius);
+                        Collider nodeCollider = NodeList[i].transform.GetComponentInChildren<Collider>();
+                        if (nodeCollider != null)
+                        {
+                            // Make the radius the largest of the object's dimensions to avoid overlap
+                            Bounds bounds = nodeCollider.bounds;
+                            NodeList[i].Radius = Mathf.Max (Mathf.Max(bounds.size.x, bounds.size.y), bounds.size.z) / 2;
+                        } else
+                        {
+                            // Make the radius a default value
+                            // TODO move this into a public field ?
+                            NodeList[i].Radius = 1f;
+                        }
+                        NodeList[i].transform.localPosition = newPos;
+                        switch (OrientType)
+                        {
+                            case OrientTypeEnum.None:
+                            default:
+                                // Don't apply any rotation
+                                break;
+
+                            case OrientTypeEnum.FaceOrigin:
+                            case OrientTypeEnum.FaceFoward:
+                                newRot = Vector3.zero;
+                                NodeList[i].transform.rotation = Quaternion.LookRotation(newRot);
+                                break;
+
+                            case OrientTypeEnum.FaceOriginReversed:
+                            case OrientTypeEnum.FaceForwardReversed:
+                                newRot = Vector3.zero;
+                                NodeList[i].transform.rotation = Quaternion.LookRotation(newRot);
+                                NodeList[i].transform.Rotate(0f, 180f, 0f);
+                                break;
+                        }
+                    }
+
+                    // Iterate [x] times
+                    // TODO move center, iterations and padding into a public field
+                    for (int i = 0; i < 100; i++)
+                    {
+                        IterateScatterPacking (NodeList, Radius);
+                    }
+                    break;
+            }
+        }
+
+        /// <summary>
+        /// Internal function for getting the relative mapping based on a source Vec3 and a radius for spherical mapping.
+        /// </summary>
+        /// <param name="source">The source <see cref="Vector3"/> to be mapped to sphere</param>
+        /// <param name="radius">This is a <see cref="float"/> for the radius of the sphere</param>
+        /// <returns></returns>
+        private Vector3 SphericalMapping(Vector3 source, float radius)
```

Comment:
radius parameter is never used.

Reply:
Is now when passed in and non-zero replaces the collections radius and is applied.